app/career/tests/test_weekly_business.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+
+from app.career.factories import WeeklyBusinessFactory
+from app.util import today
+
+
+@pytest.fixture()
+def weekly_business():
```

Comment:
I think you can put this in `conftest.py` and still use it in this test?

Reply:
jeg setter uke=5 her, som er grunnen til  at jeg ikke bruker den i `conftest`. Tror du det er en god ide å bruke de istedenfor?

Reply:
Litt opp til deg, kanskje lettere enn å jobbe med en tilfeldig uke?